keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -588,21 +814,23 @@ _This section must be completed when targeting beta graduation to a release._
   determine the health of the service?**
 
   - [x] Metrics
-    - All `errors_total` metrics below cover real errors when a Pod can't start.
+    - All `*_errors_total` metrics below cover real errors when a Pod can't start.
       - It applies to `ReadWriteOncePod` volumes when only `SELinuxMountReadWriteOncePod` feature gate is enabled.
       - It applies to all volumes when both `ReadWriteOncePod` and `SELinuxMount` feature gate is enabled.
-    - All `warnings_total` metrics below are reported when only `SELinuxMountReadWriteOncePod` feature gate is enabled and shows **future** errors that would appear after both `SELinuxMountReadWriteOncePod` and `SELinuxMount` feature gates are enabled.
+    - All `*_warnings_total` metrics below are reported when only `SELinuxMountReadWriteOncePod` feature gate is enabled and shows **future** errors that would appear after both `SELinuxMountReadWriteOncePod` and `SELinuxMount` feature gates are enabled.
       This will be evaluated in Phase 2.
-    - 1. `volume_manager_selinux_container_errors_total` + `volume_manager_selinux_container_warnings_total`: Number of errors when kubelet cannot compute SELinux context for a container.
-        This indicates an error converting SELinux context into SELinux label by github.com/opencontainers/selinux/go-selinux library.
-        Reading its source code, this should never happen, but one never knows.
-      1. `volume_manager_selinux_pod_context_mismatch_errors_total` + `volume_manager_selinux_pod_context_mismatch_warnings_total`: Number of errors when a Pod defines different SELinux contexts for its containers that use the same volume.
-         Before this feature, only one container in such a Pod could access the volume.
-         With this feature, the Pod won't even start.
-         This metric captures nr. of failed Pod starts, including periodic retries.
-      1. `volume_manager_selinux_volume_context_mismatch_errors_total` + `volume_manager_selinux_volume_context_mismatch_warnings_total`: Number of errors when a Pod uses a volume that is already mounted with a different SELinux context than the Pod needs.
-         Before this feature, both pods would start, but only one such pod could access the volume.
-         With this feature, one of the Pods won't even start.
+        1. `volume_manager_selinux_container_errors_total` + `volume_manager_selinux_container_warnings_total`: Number of errors when kubelet cannot compute SELinux label for a container.
```

Comment:
* The other metric names indicate the type of error ("..._pod_context_mismatch_...", "..._volume_context_mismatch_..."). Should we do the same here?
* Another option would be to have "volume_manager_selinux_errors_total"/"volume_manager_selinux_warnings_total" metric names and a specific `type=volume_context_mismatch|pod_context_mismatch` label but it depends if the rest of the label set is the same for all metrics.

Reply:
The labels are slightly different for each metric, depending if it is pod related (like `volume_manager_selinux_container_errors_total`) or volume related (then it has `volume_plugin` and `access_mode` label, so the admin can figure out if the issue is RWOP releated and which volume plugin / CSI driver has misconfigured `SELinuxMount` flag. I added list of labels to each metric here.